packages/flutter/test/widgets/focus_traversal_test.dart

```diff
@@ -2015,6 +2015,11 @@ void main() {
       expect(events.length, 2);
     });
 
+    testWidgets('Focus traversal does not throw when no focusable is available in a group', (WidgetTester tester) async {
+      await tester.pumpWidget(const MaterialApp(home: Scaffold(body: ListTile(title: Text('title')))));
+      await tester.sendKeyEvent(LogicalKeyboardKey.tab);
```

Comment:
Can we assert somehow that "nothing" has focus or that the focus stayed where it was even after the tab was send?

Reply:
Sure.  Done.